src/main/telemetry/telemetry.h

```diff
@@ -37,7 +37,7 @@ typedef enum {
 
 typedef struct telemetryConfig_s {
     uint8_t telemetry_switch;               // Use aux channel to change serial output & baudrate( MSP / Telemetry ). It disables automatic switching to Telemetry when armed.
-    serialInversion_e telemetry_inversion;      // also shared with smartport inversion
+    uint8_t telemetry_inversion;            // also shared with smartport inversion
```

Comment:
Sure, that can be done, but I don't think that user-control over any of the other serial options is useful (as far as I know, inversion is the only thing we want to expose).